packages/repository/src/entity/factory.test.ts

```diff
@@ -31,26 +24,41 @@ describe("Entity", () => {
       id: "super-unique-id",
       foo: "bar",
     })
+
+    vi.clearAllMocks()
   })
 
   describe("Factories", () => {
     describe("Model Factory", () => {
-      it("Given schema, When model factory called, Then return context factory", () => {
-        const result = entityModelFactory<TestRawEntityData>()
+      it("Given schema, relations, sync keys, methods, When entity model factory called, Then return entity factories", ({
```

Comment:
Czemu nie używasz Gherkina i numeracji testów? Ciężko wyłapać jak się któryś wysypie no i trochę łatwiej się czyta  z tym GIVEN, WHEN, THEN - my w BP dalej to praktykujemy :D

Reply:
Zapomniałem o tej "cudownej" praktyce :smile: Jakoś bez numeracji nadal idzie się połapać co nie działa, szczególnie jak używasz rozszerzenia do Vitest w VSC 😄 

A co do Gerhkina, to profesjonalniej robi się to przez cucumbera: https://github.com/bencompton/jest-cucumber 
Jednak narazie to chyba nie chce aż tak komplikować zapisu testów